mysite/announcements/templates/registration/login.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Login</title>
+</head>
+
+<body>
+    <form method="POST">
+        {% csrf_token %}
+        <p>
+            <label>Username</label>
+            <input type="text" name="username">
+        </p>
+        <p>
+            <label>Password</label>
+            <input type="password" name="password">
+        </p>
+        <button type="submit">Login</button>
+    </form>
```

Comment:
Can we create and use a form here too just like we created a form for registration? https://github.com/tarungarg546/annoucements_onboarding/pull/9/files#diff-a883e168e50094ccf78629dc32830eb9R3

Reply:
Yeah but here we're using views provided by Django. Django provides views for handling user authentication methods like login, logout, and password recovery. The view by default renders a template that is in registration/login.html